power_grid_model_io_native_c/power_grid_model_io_native_c/include/power_grid_model_io_native_c/vnf_pgm_converter.h

```diff
@@ -26,10 +26,9 @@ PGM_IO_API PGM_IO_VnfConverter* PGM_IO_create_vnf_converter(PGM_IO_Handle* handl
  * @brief Retrieve the transformed input data from .vnf format to PGM format
  * @param handle
  * @param converter_ptr A pointer to a PGM_IO_VnfConverter instace.
- * @param dataset A pointer to the const dataset supplied by the user.
- * @return The pointer to the const dataset instance supplied by the user which has been filled in.
+ * @return The pointer to the json string instance that holds data in PGM format.
  */
-PGM_IO_API char const* PGM_IO_get_vnf_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr);
+PGM_IO_API char const* PGM_IO_get_pgm_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr);
```

Comment:
C (not C++) does not have a type matching mechanism. as a result, if there would ever be a similar function like `PGM_IO_get_pgm_input_data(PGM_IO_Handle* handle, PGM_IO_CimConverter* converter_ptr);`, then that's not allowed (note: it is allowed and even encouraged in C++). 

instead, the name itself should be specific enough to be unique and future-proof, e.g.

```suggestion
PGM_IO_API char const* PGM_IO_vnf_get_pgm_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr);
```
or

```suggestion
PGM_IO_API char const* PGM_IO_vnf_pgm_converter_get_input_data(PGM_IO_Handle* handle, PGM_IO_VnfConverter* converter_ptr);
```